mesonbuild/dependencies/base.py

```diff
@@ -221,13 +222,17 @@ def generate_system_dependency(self, include_type: str) -> 'Dependency':
         new_dep.include_type = self._process_include_type_kw({'include_type': include_type})
         return new_dep
 
+    def get_d_features(self) -> T.Dict[str, T.Union[T.List[str], T.List['IncludeDirs']]]:
+        return self.d_features
+
```

Comment:
This isn't particularly useful, it returns a reference, but it creates the overhead of adding a function call. We can drop it and just use `self.d_features` directly